regolith/helpers/a_manurevhelper.py

```diff
@@ -100,7 +100,13 @@ def db_updater(self):
         if rc.reviewer:
             pdoc.update({'reviewer': rc.reviewer})
         else:
-            pdoc.update({'reviewer': 'sbillinge'})
+            try:
+                rc.reviewer = rc.default_user_id
```

Comment:
it doesn't look as if you updated pdoc.  Make sure you know what the code is doing and then adapt it to your needs, don't just copy-paste it....

Reply:
That was a bad mistake. It should be fixed now.

Reply:
👍